nodejs/eks/cluster.ts

```diff
@@ -144,15 +144,21 @@ export interface CoreData {
     oidcProvider?: aws.iam.OpenIdConnectProvider;
 }
 
-function createOrGetInstanceProfile(name: string, parent: pulumi.ComponentResource, instanceRoleName?: pulumi.Input<aws.iam.Role>, instanceProfileName?: pulumi.Input<string>): aws.iam.InstanceProfile {
+function createOrGetInstanceProfile(
+    name: string,
+    parent: pulumi.ComponentResource,
+    instanceRoleName?: pulumi.Input<aws.iam.Role>,
+    instanceProfileName?: pulumi.Input<string>,
+    provider?: pulumi.ProviderResource,
+): aws.iam.InstanceProfile {
     let instanceProfile: aws.iam.InstanceProfile;
     if (instanceProfileName) {
         instanceProfile = aws.iam.InstanceProfile.get(`${name}-instanceProfile`,
-            instanceProfileName, undefined, { parent: parent });
+            instanceProfileName, undefined, { parent: parent, provider });
```

Comment:
Should make arg usage consistent.
```suggestion
            instanceProfileName, undefined, { parent, provider });
```